google/cloud/bigtable/data/execute_query/metadata.py

```diff
@@ -214,6 +214,11 @@ class Bool(Type):
         type_field_name = "bool_type"
 
     class Timestamp(Type):
+        """
+            Timestamp supports :class:`DatetimeWithNanoseconds` but Bigtable SQL does
+            not currently support nanoseconds precision. We support this for potential
+            compatibility in the future. Nanoseconds are currently ignored.
+        """
```

Comment:
It looks like this is what needs to change:

```suggestion
        """
        Timestamp supports :class:`DatetimeWithNanoseconds` but Bigtable SQL does
        not currently support nanoseconds precision. We support this for potential
        compatibility in the future. Nanoseconds are currently ignored.
        """
```